go/chat/attachment.go

```diff
@@ -114,7 +129,7 @@ func putSingle(ctx context.Context, log logger.Logger, r io.Reader, size int64,
 	// attachment.  But putSingle is only called for attachments <= 5MB, so
 	// this isn't horrible.
 	buf := make([]byte, size)
-	n, err := r.Read(buf)
+	n, err := io.ReadFull(r, buf)
```

Comment:
Wow my `ReadFull` comment above ended up totally in the wrong place, sorry about that. I meant here. I think the `int64(n) != size` check on line 136 below is now impossible?